packages/wrangler-devtools/README.md

```diff
@@ -1,6 +1,79 @@
 # Wrangler Devtools Pages Project
 
-This package contains a Workers specific version of Chrome Devtools that is used by the Wrangler dev command.
+This package contains a Workers specific version of Chrome Devtools that is used by the Wrangler dev command. It is a customized fork of Chrome DevTools specifically tailored for debugging Cloudflare Workers. This package provides Worker-specific functionality through carefully maintained patches on top of Chrome DevTools.
+
+## Overview
+
+This package is used across multiple Cloudflare products:
+
+- Workers Playground (`workers-playground`)
+- Quick Editor (`@cloudflare/quick-edit`)
+- Wrangler CLI via the `InspectorProxy`
+
+## Features
+
+Our customized DevTools implementation provides:
+
+- Source code viewing and live updates
+- Network request inspection
+- Worker-specific UI optimizations
+
+## Development
+
+We maintain this fork by applying patches on top of Chrome DevTools. These patches need to be periodically rebased as Chrome DevTools evolves.
+
+**Key Development Tasks:**
+
+- Generating patches from our customizations
+- Rebasing patches onto new Chrome DevTools versions
+- Testing functionality across all integration points
+
+## Updating DevTools
+
+We perform quarterly updates to stay current with upstream Chrome DevTools. The update process involves:
+
+1. Cloning the devtools-frontend repo
+2. Applying our existing patches
+3. Rebasing onto the latest Chrome DevTools
+4. Regenerating patches
+5. Thorough testing across all integration points
+
+**For detailed instructions on updating DevTools, please refer to our internal documentation at:
+https://wiki.cfdata.org/display/WX/Keeping+DevTools+up-to-date**
```

Comment:
I am not sure if we should link the internal wiki here. Do we have other examples doing this? How about moving the instructions here and put a github link in the wiki?

Reply:
Oh yeah, you're absolutely right. Not sure why I put this here. I'll just say to refer to the internal documentation.

Reply:
there's definitely a couple floating around if you search `wiki.cfdata`, should we clean up those...?

Reply:
Yeap, I think it makes sense to get rid of those.